cardigan/stories/components/AudioPlayerNew/AudioPlayerNew.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ReadmeDecorator } from '@weco/cardigan/config/decorators';
+import AudioPlayer from '@weco/content/components/AudioPlayerNew/AudioPlayer';
+import Readme from '@weco/content/components/AudioPlayerNew/README.mdx';
+
+const meta: Meta<typeof AudioPlayer> = {
+  title: 'Components/AudioPlayerNew',
+  component: AudioPlayer,
+  args: {
+    audioFile:
+      'https://iiif.wellcomecollection.org/av/b2248887x_0001.wav/full/max/default.mp3#identity',
+    title: 'Mat Fraser: interview 1',
+    isDark: true,
+  },
```

Comment:
Suggestion;
<img width="460" alt="Screenshot 2025-04-10 at 13 13 37" src="https://github.com/user-attachments/assets/d7449d5c-a7d4-4f82-8409-88b394e5195d" />

Transcript to be a boolean and we add a mock one?
`titleProps` disabled
`idPrefix` is actually unused so fully removed?

Reply:
I hadn't even noticed that `transcript` was a prop to the component. And tbh I'm not sure it should be – we're not using it on the guide stop pages ([e.g.](https://github.com/wellcomecollection/wellcomecollection.org/blob/0fbe81b1208f74c7c21c64006a175cc0bdddf794/content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx#L367-L373)). I'll see if we do use it anywhere or if we can remove it altogether.

`idPrefix` should be removed, you're right - will do that.

And yeah `titleProps` aren't useful for Cardigan – I'll disable them.

Reply:
I've added one to the story and hidden it in the argTypes.